cogs/helper.py

```diff
@@ -1,40 +1,89 @@
 import discord
 from discord.ext import commands
+from utils.config import subjects 
 
 
 class Helper(commands.Cog):
     def __init__(self, bot: commands.Bot) -> None:
-        self.bot = bot
-
+        self.bot = bot        
+        self.subjects_list = [pair.split(",") for pair in subjects.split(";")]
+        self.subject_channels_list = [int(pair[0]) for pair in self.subjects_list]
```

Comment:
I think you could store this data in a dictionary that maps from channel IDs to role IDs.
- `self.subjects.keys()` would return a collection of channel IDs.
- `self.subjects.values()` would return a collection of role IDs.
- `self.subjects[channel_id]` would return the corresponding role ID.

This would make the context menu code more readable:
- `interaction.channel.id not in self.subjects` would work (since membership is tested across keys with this syntax).
- `subject_helper_role = self.subjects[interaction.channel.id]` retrieves the desired role ID.